### ISSUE TITLE
Photo sorting configuration

### ISSUE THREAD

**Issue by JaviSoto** [sambecker/exif-photo-blog#29]
Hi! I really like this project. I switched my personal website over from Squarespace and I’m really happy with the result :)

One thing I haven’t been able to figure out even looking through the codebase is how the order of photos is controlled. I tried using the “priority order” setting in each photo, but it doesn’t seem to have any effect, whether I use a low or high value. 

What am I missing?

Thank you and keep up the great work!

**Comment by sambecker** [comment]
Thank you for your kind words! This is definitely something we can look into.

`priority_order` was added to have control over the auto-generated template urls (`/template-image`, `/template-image-tight`) but I think it can be generalized.

Which views would you want sorted by `priority_order`? Anything beyond the home page (`/` + `/grid`)?

**Comment by JaviSoto** [comment]
Yeah that's the one I was interested in, the main grid in the main page. Setting the priority order of a photo doesn't seem to have any effect for me. How could I help debug this?

**Comment by sambecker** [comment]
Ok—makes sense. Let me put up a PR to test out.

**Comment by sambecker** [comment]
@JaviSoto can you try #30 and see if it behaves as expected? LMK!

**Comment by sambecker** [comment]
Hi @JaviSoto popping back in here to you let you sorting has been totally rethought via #277 + #283.

Make sure to sync your fork for the latest features.

To make sure the priority field is taken into account, enable:
`NEXT_PUBLIC_PRIORITY_BASED_SORTING=1`

To offer full sort menu controls, enable:
`NEXT_PUBLIC_NAV_SORT_CONTROL=menu`

<img width="644" height="460" alt="Image" src="https://github.com/user-attachments/assets/6bdcca7d-0bd0-42c2-a6c3-ea47acfc457f" />
